### ISSUE TITLE
Software download issues

### ISSUE THREAD

**Issue by pwh124** [nanoporetech/dorado#28]
Hello,

The link address for software download works when I click to download but does not work with wget.

I would like to be able to download straight to our server if possible.  Linux address is shown as "https://nanoporetech.box.com/shared/static/h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz"

Paul

**Comment by iiSeymour** [comment]
Hey @pwh124 the links are direct links and do work with `wget`.

```
% wget https://nanoporetech.box.com/shared/static/h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz
--2022-10-05 18:45:25--  https://nanoporetech.box.com/shared/static/h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz
Resolving nanoporetech.box.com (nanoporetech.box.com)... 74.112.186.144
Connecting to nanoporetech.box.com (nanoporetech.box.com)|74.112.186.144|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: /public/static/h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz [following]
--2022-10-05 18:45:25--  https://nanoporetech.box.com/public/static/h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz
Reusing existing connection to nanoporetech.box.com:443.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://nanoporetech.ent.box.com/public/static/h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz [following]
--2022-10-05 18:45:26--  https://nanoporetech.ent.box.com/public/static/h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz
Resolving nanoporetech.ent.box.com (nanoporetech.ent.box.com)... 74.112.186.144
Connecting to nanoporetech.ent.box.com (nanoporetech.ent.box.com)|74.112.186.144|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://ec2euc1.boxcloud.com/d/1/a1!8CEmmVQuFQ144DDmC_3bSwoc_aqltLWy_4OkBSCdBHD102gWdBNUjfh0A5Ad8HKwHQ5--n6zfJiJs9XThKlKqgF-IVAm_8niuHSms0BFuXIfmwkSHGKK6188GP4Q2038qyLZJ2klAtZ1Y6w-N1HSMQEFGfYKx4qldfxa8ACqc4jCDo9fHYKlfA4gIKS3tKcQeEl0XiVgK66YwUu8ywzwgetqHbGpduIQjNZMme1kIaYN9NUmaJSr0KPFKr6zS9rkmZ7TPWKXEv_qv1xO9wdkhIODp51Euwzj3mkFH8Du3N6YdOiYYT2bmtUK-5O5cQCIVAD2xyZHiZK7jKe6v21SegVRMFtXqvGJ38Ecgd8QN5xVjzIXph-A0mJxjZ2UF36JVlD5UFQLfN-ygCXZ1MCBNzxsKhm9DzGxgFjLXXsanJfgv3zWc6JCJbZwEnNQkQDqAE6_fCd6FfiFjyJQXpZZmG3zgq-nm-ekeSIzixwIr99Gitjxnhb2oHTYvORJhGnVCwUfC_yfKozqbuQ5C0s61xpndIIRBhQktYEV0yuhEKOhMsWQoi_LOalpPwaKeAeDuLPs56F3fi9UgjwXWHC9SCPmoPFSeqTb94nTCWZ7ptb8ZsBErLJi5M5lsmAfcc1FrmgDxCiP_bUQN7lrsU6LCp7wcH7hMLH2thnrLmhRyoCHypMdmQJZ_bNG4m9RaOWeGH8RpE8i0127qFLp52BmwqVmd-4B1yOpTswzulVr0x-4X1xgJjuUJGmq20K1EGZbiHqu5S0iPp40-liqwqryWJ0LPo1jOXrhvzkt4GcvCtQ3X9KLi_i7nKz3DuzD0nB5qRcHiVAXceQHRuxOwVdRDcVzqgMA6yYocpPwUHVLyP-sOJECuylrbUKY9M38Ieeo2WdCAnI5ws9hYTxvacn8HNPUNePaoTgE20MsDfOoKT5jT8r_UiJYhjPq_ipSVBQwVxqRc67J8Yz2ZTQyXs1OBmVwVP50CZgSXWyHz_SK8oILaBIDyFrq65JsMFJ7ZAa_TykDzRfuX81E5_qBk26Op1synYXE0x3lENlN5NrExO_9Ri-70SDr6wDva3WfeHf_83Yv1BX1uRs5gG-FPxqDqzfzJkdsA-nWy9ziM0_q_DWAJzcnEfe5DtkAgmOFnhbKGs3XMyGpmoj7ocwYpFplueHgi5DQZulnGJG-48YnqxOgnHmFS2mWUAepFlfYJzsR2YyvwFRAVCF8HKtUQCpPB_B1NxK8rf3l5acZM2TLF03GibFYuvuHxzMJamQD9H-EixJhIWh1_iqeYPGhMkzbeLv__p_ZBP9D8GDo8R7EWN5RA_o2bAhiRR5xBlMpV4ZGtIKqdwAHwB2aWHoZWzA3aT1k-22VmgSHTATVi0iqjFq2eMKLgbDSr7aYLlrNDloQmw9AYWKaz_kyqHBlykCVsEw_zrXAveTs_U1t7sClpe06UIgYahMUImtNnKMOzVltbY1LPBxhNva4v826OQcX9Ursq4jesg7tavdz/download [following]
--2022-10-05 18:45:27--  https://ec2euc1.boxcloud.com/d/1/a1!8CEmmVQuFQ144DDmC_3bSwoc_aqltLWy_4OkBSCdBHD102gWdBNUjfh0A5Ad8HKwHQ5--n6zfJiJs9XThKlKqgF-IVAm_8niuHSms0BFuXIfmwkSHGKK6188GP4Q2038qyLZJ2klAtZ1Y6w-N1HSMQEFGfYKx4qldfxa8ACqc4jCDo9fHYKlfA4gIKS3tKcQeEl0XiVgK66YwUu8ywzwgetqHbGpduIQjNZMme1kIaYN9NUmaJSr0KPFKr6zS9rkmZ7TPWKXEv_qv1xO9wdkhIODp51Euwzj3mkFH8Du3N6YdOiYYT2bmtUK-5O5cQCIVAD2xyZHiZK7jKe6v21SegVRMFtXqvGJ38Ecgd8QN5xVjzIXph-A0mJxjZ2UF36JVlD5UFQLfN-ygCXZ1MCBNzxsKhm9DzGxgFjLXXsanJfgv3zWc6JCJbZwEnNQkQDqAE6_fCd6FfiFjyJQXpZZmG3zgq-nm-ekeSIzixwIr99Gitjxnhb2oHTYvORJhGnVCwUfC_yfKozqbuQ5C0s61xpndIIRBhQktYEV0yuhEKOhMsWQoi_LOalpPwaKeAeDuLPs56F3fi9UgjwXWHC9SCPmoPFSeqTb94nTCWZ7ptb8ZsBErLJi5M5lsmAfcc1FrmgDxCiP_bUQN7lrsU6LCp7wcH7hMLH2thnrLmhRyoCHypMdmQJZ_bNG4m9RaOWeGH8RpE8i0127qFLp52BmwqVmd-4B1yOpTswzulVr0x-4X1xgJjuUJGmq20K1EGZbiHqu5S0iPp40-liqwqryWJ0LPo1jOXrhvzkt4GcvCtQ3X9KLi_i7nKz3DuzD0nB5qRcHiVAXceQHRuxOwVdRDcVzqgMA6yYocpPwUHVLyP-sOJECuylrbUKY9M38Ieeo2WdCAnI5ws9hYTxvacn8HNPUNePaoTgE20MsDfOoKT5jT8r_UiJYhjPq_ipSVBQwVxqRc67J8Yz2ZTQyXs1OBmVwVP50CZgSXWyHz_SK8oILaBIDyFrq65JsMFJ7ZAa_TykDzRfuX81E5_qBk26Op1synYXE0x3lENlN5NrExO_9Ri-70SDr6wDva3WfeHf_83Yv1BX1uRs5gG-FPxqDqzfzJkdsA-nWy9ziM0_q_DWAJzcnEfe5DtkAgmOFnhbKGs3XMyGpmoj7ocwYpFplueHgi5DQZulnGJG-48YnqxOgnHmFS2mWUAepFlfYJzsR2YyvwFRAVCF8HKtUQCpPB_B1NxK8rf3l5acZM2TLF03GibFYuvuHxzMJamQD9H-EixJhIWh1_iqeYPGhMkzbeLv__p_ZBP9D8GDo8R7EWN5RA_o2bAhiRR5xBlMpV4ZGtIKqdwAHwB2aWHoZWzA3aT1k-22VmgSHTATVi0iqjFq2eMKLgbDSr7aYLlrNDloQmw9AYWKaz_kyqHBlykCVsEw_zrXAveTs_U1t7sClpe06UIgYahMUImtNnKMOzVltbY1LPBxhNva4v826OQcX9Ursq4jesg7tavdz/download
Resolving ec2euc1.boxcloud.com (ec2euc1.boxcloud.com)... 18.194.95.21, 18.194.95.20
Connecting to ec2euc1.boxcloud.com (ec2euc1.boxcloud.com)|18.194.95.21|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1855860162 (1.7G) [application/octet-stream]
Saving to: ‘h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz’

h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz            100%[====================================================================================================>]   1.73G  24.8MB/s    in 71s     

2022-10-05 18:46:38 (24.9 MB/s) - ‘h8eqc9htxk938jzpl4fch2rqlm48yeb0.gz’ saved [1855860162/1855860162]
```

Maybe your version of `wget` isn't following redirects or your network is intercepting? What exactly do you see?

**Comment by pwh124** [comment]
`wget` successfully downloads the ".gz" file. So I tried to `gunzip` it, leaving me with "h8eqc9htxk938jzpl4fch2rqlm48yeb0" to which I could not move into. 

When I ran `file h8eqc9htxk938jzpl4fch2rqlm48yeb0`:
`h8eqc9htxk938jzpl4fch2rqlm48yeb0: POSIX tar archive`

`mkdir dorado & tar -xvf h8eqc9htxk938jzpl4fch2rqlm48yeb0 -C dorado` completely decompresses the dorado directory then.

I think renaming the link with ".tar.gz" would clear this up. Thanks for your help!
